### PR TITLE
Update glossary: Add self-test

### LINKED ISSUE #3000: Missing Glossary entry for self-test
## Where to find the issue

- https://www.coronawarn.app/en/faq/#glossary
- https://www.coronawarn.app/de/faq/#glossary

## Describe the issue

The [Glossary](https://www.coronawarn.app/de/faq/#glossary) contains no entry for a self-test.

Test type | Glossary entry
--- | ---
PCR-Test | https://www.coronawarn.app/de/faq/results/#glossary_pcr_test
PoC-NAT-Test<br>(PCR-Schnelltest) | https://www.coronawarn.app/de/faq/results/#glossary_rapid_pcr_test
Antigen-Schnelltest | https://www.coronawarn.app/de/faq/results/#glossary_rapid_antigen_test
Selbsttest | **missing**

## Suggested change

1. Add a Glossary entry `#glossary_selftest` and use for instance existing BMG information (see below) to write a description of lateral flow self-tests in English and German.
2. Extend the FAQ article https://www.coronawarn.app/en/faq/results/#rat_tan_hotline "Can I share my positive rapid antigen test or rapid PCR test result via the TAN hotline?" to link to the new glossary entry.

## Background information

[Was ist der Unterschied zwischen PCR-Tests, PoC-NAT-Tests, Antigen-Schnelltests und Selbsttests?](https://www.bundesgesundheitsministerium.de/coronavirus/nationale-teststrategie/faq-covid-19-tests.html#c20484)

"Antigen-Selbsttests haben ihren Namen, weil diese Tests jeder selber, beispielsweise zuhause, machen kann. Selbsttests sind zur Anwendung durch Privatpersonen bestimmt und weisen wie auch Antigen-Schnelltests virale Eiweiße nach. Dafür muss die Gebrauchsanweisung leicht und verständlich sein, sowie Probenentnahme und -auswertung entsprechend einfach sein. Bei korrekter Abstrichnahme sind Ergebnisse von Selbsttests genauso verlässlich wie durch geschultes Personal vorgenommene Antigen-Schnelltests."

[Welche Tests sind wofür geeignet?](https://www.bundesgesundheitsministerium.de/coronavirus/nationale-teststrategie/faq-covid-19-tests.html#c23573)

"Antigen-Selbsttests können zusätzliche Sicherheit in konkreten Situationen im Alltag geben – insbesondere auch da, wo sich Menschen möglicherweise ohne Maske begegnen – etwa bei einem privaten Besuch oder einer Feier. Sie können auch im Rahmen der Testkonzepte der Länder in Schulen und Kitas eingesetzt werden."

[Selbsttest für den Nachweis des Coronavirus](https://www.zusammengegencorona.de/testen/selbsttest-fuer-den-nachweis-des-coronavirus-sars-cov-2/)

"Selbsttests sind freiverkäuflich und können auch von ungeschulten Personen sicher angewendet werden. Für die Auswertung eines Selbsttests braucht es auch kein Labor. Sie müssen sich also nicht eigens zu einer Ärztin beziehungsweise einem Arzt, dem Gesundheitsamt oder einer Teststation begeben, sondern können sich bequem zuhause selbst testen. Hinzu kommt, dass die kostenlosen Antigen-Schnelltests – die sogenannten Bürgertests – seit dem 30. Juni auf vulnerable und weitere bestimmte Personengruppen eingeschränkt sind. ..."

---
Internal Tracking ID: [EXPOSUREAPP-13526](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13526)

### PR DESCRIPTION
**Pull request by larswmh**
Resolves #3000 

[#glossary_selftest](https://www.coronawarn.app/en/faq/results/#glossary_selftest)
image
image

[#glossary_rapid_antigen_test](https://www.coronawarn.app/en/faq/results/#glossary_rapid_antigen_test)
image
image

[#rat_tan_hotline](https://www.coronawarn.app/en/faq/results/#rat_tan_hotline)
image
image

---
Internal Tracking ID: [EXPOSUREAPP-13526](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13526)

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -3362,6 +3362,11 @@
...
                     "description": "The Corona-Warn-App Science Blog covers various scientific topics related to the Corona-Warn-App, such as its effectiveness, benefits, as well as results of scientific evaluations conducted by the Robert Koch Institute. See <a href='../../science'>Science-Blog</a>."
                 },
+                {
+                    "term": "Self-test",
+                    "anchor": "selftest",
+                    "description": "Self-tests are available free of charge and can be used safely even by untrained individuals. For the evaluation of a self-test, which will be done within 15 to 30 minutes, there is no need for a laboratory. Therefore, you do not have to go to a doctor, the health department or a testing station, but can test yourself comfortably at home. However, a self-test is only reliable if it has been carried out correctly. Because this is difficult to detect, self-test results are not sufficient for testing yourself free from a prescribed quarantine, nor for confirmation of infection. In both cases, a certified <a href='#glossary_rapid_antigen_test'>rapid antigen test</a> is necessary."
```
Comment (Ein-Tim) [inline src/data/faq.json:3368]: I'm not sure if "free of charge" is the correct term to use here, as you have to buy these tests and they are not for free.

**Review by Ein-Tim — COMMENTED** [review]
One in-line comment.

File: src/data/faq.json (hunk 1 above)
Comment (MikeMcC399) [reply src/data/faq.json:3368]: The translation for "freiverlkäuflich" would be "freely available", not "free of charge"

Comment (larswmh) [reply src/data/faq.json]: Thanks for the information. It has been changed to "freely available" in https://github.com/corona-warn-app/cwa-website/pull/3001/commits/6fae75ec81cc132a4e6e6f29927df3283fa16916

**Comment by MikeMcC399** [comment]
I suggest also to link "self-test" in `#glossary_rapid_antigen_test` to the new `#glossary_selftest`.

**Comment by larswmh** [comment]
> I suggest also to link "self-test" in `#glossary_rapid_antigen_test` to the new `#glossary_selftest`.

Thanks for the suggestion. A link from `#glossary_rapid_antigen_test` to `#glossary_selftest` has been added in https://github.com/corona-warn-app/cwa-website/pull/3001/commits/81561b08e4f3947d5bdc221a641b6f88af785531.

**Review by MikeMcC399 — APPROVED** [review]
@larswmh 

Thanks for implementing this request!

**Review by dsarkar — APPROVED** [review]

**Review by GisoSchroederSAP — APPROVED** [review]
If possible: change the phrase
"testing yourself free"
into 
- either to: "testing yourself to release"
- or "testing to free yourself"

Other than that - content and wording approved.

**Pull request merged**